Cargo.toml

```diff
@@ -136,7 +136,7 @@ codec = { path = "components/codec" }
 tipb_helper = { path = "components/tipb_helper" }
 tipb = { git = "https://github.com/pingcap/tipb.git" }
 # TODO: use master branch after the next version is released.
-kvproto = { git = "https://github.com/pingcap/kvproto.git", branch = "release-3.0" }
+kvproto = { git = "https://github.com/pingcap/kvproto.git", rev = "3989027cb1ea74a513205d84adbce69d7e46b97d" }
```

Comment:
em, can we still use release-3.0?

Reply:
Yes, as long as we merge https://github.com/pingcap/kvproto/pull/409 first.